src/binary/ibc-setup/commands/channels.spec.ts

```diff
@@ -0,0 +1,278 @@
+import fs from 'fs';
+import os from 'os';
+
+import test from 'ava';
+import sinon from 'sinon';
+
+import { State as ChannelState } from '../../../codec/ibc/core/channel/v1/channel';
+import { Order } from '../../../codec/ibc/core/channel/v1/channel';
+import { ChannelPair, Link } from '../../../lib/link';
+import { TestLogger } from '../../../lib/testutils';
+import { Logger } from '../../create-logger';
+import { signingClient } from '../../utils/signing-client';
+
+import { simappChain, wasmdChain } from './chains';
+import { channelStateAsText, Options, run } from './channels';
+
+const fsReadFileSync = sinon.stub(fs, 'readFileSync');
+const consoleLog = sinon.stub(console, 'log');
+
+const mnemonic =
+  'enlist hip relief stomach skate base shallow young switch frequent cry park';
+
+const registryYaml = `
+version: 1
+
+chains:
+  local_wasm:
+    chain_id: testing
+    prefix: wasm
+    gas_price: 0.025ucosm
+    rpc:
+      - http://localhost:26659
+  local_simapp:
+    chain_id: simd-testing
+    prefix: cosmos
+    gas_price: 0.025umuon
+    rpc:
+      - http://localhost:26658`;
+
+test.beforeEach(() => {
+  sinon.reset();
+});
+
+let channel: ChannelPair;
+let link: Link;
+
+test.before(async () => {
+  const ibcClientSimapp = await signingClient(simappChain, mnemonic);
+  const ibcClientWasm = await signingClient(wasmdChain, mnemonic);
+  link = await Link.createWithNewConnections(ibcClientWasm, ibcClientSimapp);
+  channel = await link.createChannel(
+    'A',
+    'transfer',
+    'custom',
+    Order.ORDER_UNORDERED,
+    'ics20-1'
```

Comment:
Are these values not available from test utils?

Reply:
I didn't know but there's something. I had to flip things up but it works: https://github.com/confio/ts-relayer/pull/153/commits/c47a93696d3dcf62f3c4c403c3f77f7b9ca0f34a